dlrs/main/classes/ManageLookupRollupSummariesNewController.cls

```diff
@@ -0,0 +1,364 @@
+public with sharing class ManageLookupRollupSummariesNewController {
+    public LookupRollupSummary2__mdt LookupRollupSummary {get;set;}
+    Set<String> rFields {get;set;}
+    public boolean isMultiSelectRField {set;get;}
+    public String selectedRelationFieldName {get;set;}
+    public String selectedLookup {get;set;}
+    public Boolean MetadataConnectionError {get;set;}       
+    public String selectedParentObject {get;set;}
+    public String selectedChildObject {get;set;}
+    public String selectedParentField {get;set;}
+    public String selectedChildField {get;set;}
+    public String selectedRField {get;set;}
+    public List<SelectOption> parentObjFields {get;set;}
+    public List<SelectOption> childObjectList {get;set;}
+    public List<SelectOption> childObjFields {get;set;}
+    public boolean isParentObjectSelected {get;set;}
+    public boolean isChildObjectSelected {get;set;}
+    //public Map<String,String> objectWithRelationshipField {get;set;}
+    public Map<String, List<String>> PnameWithRField {get;set;}
```

Comment:
I'd prefer to remove commented code. 
This goes for all comments in this class. 
Since we haven't agreed on these code style questions, I won't reject the PR.

Reply:
The rationale is the usual one: Code moves forward, comments don't. After a while the code in the comments becomes undeployable because the rest moved. 
it detracts from the actual code logic etc etc